src/main/java/org/codehaus/mojo/servicedocgen/ServiceDocGenReport.java

```diff
@@ -108,6 +112,12 @@
     @Parameter( defaultValue = "false" )
     private boolean introspectFields;
 
+    @Parameter( defaultValue = "index.html, openapi.yaml" )
+    private String outputName;
+
+    @Parameter( defaultValue = "html, openapi_yaml" )
+    private String reportType;
```

Comment:
why do we need this at all?
After all we just instantiate a template via velocity. Why do we need to create an Enum with a constant for each template file?